Firestore/Example/App/macOS/AppDelegate.m

```diff
@@ -26,29 +26,6 @@ @interface AppDelegate ()
 @implementation AppDelegate
 
 - (void)applicationDidFinishLaunching:(NSNotification *)aNotification {
-  // create a firestore db
-  NSString *filePath = [[NSBundle mainBundle] pathForResource:@"GoogleService-Info"
```

Comment:
Was this accidentally committed previously?

Reply:
This was committed previously because it was the only way to validate that Firestore even worked on macOS. I believe we'll have this covered independently:

  * unit tests (previous PR)
  * enabling on Travis (next)
  * integration tests (next after that)